ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/blocks/BeaconBlock.java

```diff
@@ -91,7 +93,7 @@ public boolean equals(Object obj) {
     }
 
     BeaconBlock other = (BeaconBlock) obj;
-    return slot == other.getSlot()
+    return Objects.equals(this.getSlot(), other.getSlot())
```

Comment:
Why use this syntax over `this.getSlot().isEquals(other.getSlot())`

Reply:
I just wanted to make sure it is in line with other equals statements